claudio-plugin/skills/google-drive-reader/scripts/list_files.sh

```
@@ -0,0 +1,145 @@
+
+    if [ -n "$FOLDER_ID" ]; then
+        parts+=("'${FOLDER_ID}' in parents")
+    fi
+
+    if [ -n "$TYPE_FILTER" ]; then
+        local mime
+        mime=$(resolve_mime_type "$TYPE_FILTER")
+        parts+=("mimeType = '${mime}'")
+    fi
+
+    local IFS=" and "
```

Shellcheck (0.11.0)

[warning] This IFS value contains the literal letter 'n'. For tabs/linefeeds/escapes, use $'..', literal, or printf.

(SC2141)